python/nmigen_cfu/cfu.py

```diff
@@ -200,6 +192,29 @@ def __init__(self, instructions):
             self.reset
         ]
 
+    def elab_instructions(self, m):
+        """Make instructions this CFU will execute.
+
+        Returns:
+          A dictionary with keys in range(8) containing the CFU's instructions.
```

Comment:
Is "range(8)" because only funct3 is used for this?

Reply:
Ah, yes, I see it in the comments above.   I hadn't been aware before.

Reply:
Thanks, Tim.